simcon_project/conversation_templates/models/template_node_choice.py

```diff
@@ -10,12 +10,12 @@ class TemplateNodeChoice(models.Model):
     id: Primary key id for TemplateNodeChoice object
     choice_text: A text description for the Choice object
     destination_node: The TemplateNode object that a Choice object leads to
-    parent_template: The TemplateNode object that a Choice object belongs to
+    parent_template_node: The TemplateNode object that a Choice object belongs to
     """
     id = models.UUIDField(unique=True, editable=False, primary_key=True, default=uuid.uuid4)
     choice_text = models.CharField(max_length=500)
     destination_node = models.ForeignKey('conversation_templates.TemplateNode', default=0, null=True, blank=True, related_name='parent_choices', on_delete=models.CASCADE)
-    parent_template = models.ForeignKey('conversation_templates.TemplateNode', default=0, related_name='choices', on_delete=models.CASCADE)
+    parent_template_node = models.ForeignKey('conversation_templates.TemplateNode', default=0, related_name='choices', on_delete=models.CASCADE)
```

Comment:
This change isn't being reflected in some files (conversations.py is the only one I know of right now). Navigating through the conversation as a student isn't possible right now since the name has been changed.

Reply:
Just ignore this comment. Seems like the changes are reflected in the code, but when I run it its not registering the changes for some reason.